CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
 
 ## master
 
+- [PR [#87](https://github.com/palkan/logidze/pull/87)] Adding debounce time to avoid spamming changelog creation ([@zocoi][])
+
+Usage:
+
+```
+# 5000ms
+rails generate logidze:model story --debounce_time=5000
+```
+
+You see the following in generated migration
+
+```sql
+CREATE TRIGGER logidze_on_stories
+      BEFORE UPDATE OR INSERT ON stories FOR EACH ROW
+      WHEN (coalesce(#{current_setting('logidze.disabled')}, '') <> 'on')
+      EXECUTE PROCEDURE logidze_logger(null, 'updated_at', null, 5000);
+```
+
+How to upgrade.
+
+Please run `rails generate logidze:install --update` to regenerate stored functions.
+
+This feature checks if several logs came in within a debounce time period then only keep the latest one
+by merging the latest in previous others.
+
+The concept is similar to https://underscorejs.org/#debounce
+
+without `debounce_time`
+
+```
```

Comment:
Add `js` lang attributes here and in the snippet below to make RuboCop ignore these snippets (right it considers them as Ruby code)